qutebrowser/browser/webengine/notification.py

```diff
@@ -360,6 +360,7 @@ def _on_adapter_clicked(self, notification_id: int) -> None:
             log.misc.debug("Did not find matching notification, ignoring")
             return
 
+        self._focus_first_matching_tab(notification)
```

Comment:
Is it possible to do this after the `notification.click()` below? Then we could have a `return` after the workaround logging, to avoid the crash here. I don't think there's a way to implement support for this for old PyQt versions, because we can't call `notification.origin()`.